packages/blade/jest.native.config.js

```diff
@@ -29,7 +29,8 @@ module.exports = {
   moduleNameMapper: {
     '^\\~src/(.*)': '<rootDir>/src/$1',
     '^\\~components/(.*)': '<rootDir>/src/components/$1',
-    '^\\~utils': '<rootDir>/src/utils',
+    '^\\~utils$': '<rootDir>/src/utils',
+    '^\\~utils/(.*)': '<rootDir>/src/utils/$1',
```

Comment:
is this required? it's possible to import `~utils/something` without this as well right?

Reply:
I tried without as well, I think it is not able to resolve imports without these changes. Same goes for `jest.web.config.js`